src/libraries/System.Text.Json/tests/JsonNode.TraversalTests.cs

```diff
@@ -279,5 +279,26 @@ void CheckNestedValues(JsonObject jsonObject)
             Assert.IsType<JsonArray>(jsonObject["array"]);
             Assert.IsType<JsonNumber>(nestedObject["array"]);
         }
+
+        [Fact]
+        public static void TestInvalidJsonNodeOptions()
```

Comment:
Can you also add a test for what the values of a default `JsonNodeOptions` are?

So, assert the values of the options and that `JsonNodeOptions options = default;` and `var options = new JsonNodeOptions()` are equal for each of the options.

Reply:
Absolutely. I'll add it soon.